Libraries/PeriphDrivers/Source/TPU/tpu_reva.c

```diff
@@ -261,7 +261,8 @@ int MXC_TPU_RevA_Cipher_Config(mxc_tpu_reva_regs_t *tpu, mxc_tpu_reva_modesel_t
 // ******************************* Function to Select the Source of the Cipher Key *************************************
 int MXC_TPU_RevA_Cipher_KeySelect(mxc_tpu_reva_regs_t *tpu, mxc_tpu_reva_keysrc_t key_src)
 {
-    MXC_SETFIELD(tpu->cipher_ctrl, MXC_F_TPU_REVA_CIPHER_CTRL_SRC, key_src);
+    MXC_SETFIELD(tpu->cipher_ctrl, MXC_F_TPU_REVA_CIPHER_CTRL_SRC,
+                 key_src << MXC_F_TPU_REVA_CIPHER_CTRL_SRC_POS);
```

Comment:
[mxc_tpu_reva_keysrc_t](https://github.com/analogdevicesinc/msdk/blob/main/Libraries/PeriphDrivers/Source/TPU/tpu_reva.h#L53) mapped with set values ( _S_ )  instead of value ( _V_ ) so that key_src not need to be shifted.

https://github.com/analogdevicesinc/msdk/blob/main/Libraries/PeriphDrivers/Source/TPU/tpu_reva_regs.h#L213

Reply:
https://github.com/analogdevicesinc/msdk/pull/1104/commits/851de155d803be14456a7fea99579fdb43c81c99